{
  "contest": 420,
  "handle": "0xAkira",
  "risk": "3",
  "title": "Function call error",
  "issueId": 355,
  "issueUrl": "https://github.com/code-423n4/2024-07-traitforge-findings/issues/355",
  "originalIssueId": 1230,
  "originalIssueUrl": "https://github.com/code-423n4/2024-07-traitforge-validation/issues/1230"
}